MultimodalQnA/ui/gradio/conversation.py

```diff
@@ -49,9 +48,9 @@ def get_prompt(self):
             # Need to do RAG. If the query is text, prompt is the query only
             if self.audio_query_file:
                 ret = [{"role": "user", "content": [{"type": "audio", "audio": self.get_b64_audio_query()}]}]
-            elif self.image_query_file:
-                b64_image = get_b64_frame_from_timestamp(self.image_query_file, 0)
-                ret = [{"role": "user", "content": [{"type": "text", "text": self.messages[0][1]},{"type": "image_url", "image_url": {"url": b64_image}}]}]
+            elif len(messages) in self.image_query_files:
```

Comment:
I don't think an audio file and an image file in an initial query should be mutually exclusive. You should be able to send both in a single initial query.